File: app/src/main/java/vergecurrency/vergewallet/helpers/transaction/TransactionHeaderItem.java
package vergecurrency.vergewallet.helpers.transaction;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import com.google.android.material.snackbar.Snackbar;

import org.w3c.dom.Text;

import vergecurrency.vergewallet.R;

public class TransactionHeaderItem implements TransactionItem {
    private final String name;

    public TransactionHeaderItem(String name) {
        this.name = name;
    }

    @Override
    public int getViewType() {
        return TransactionRowType.HEADER_ITEM.ordinal();
    }

    @Override
    public View getView(LayoutInflater inflater, View convertView, ViewGroup parent, int position) {
        View view;
        if (convertView == null) {
            view = inflater.inflate(R.layout.listview_item_transactions_header, null);
        } else {
            view = convertView;
        }
        TextView txv = view.findViewById(R.id.listview_transactions_header_title);
        txv.setText(name);
        return view;
    }

}

File: app/src/main/kotlin/vergecurrency/vergewallet/view/ui/components/transaction/TransactionHeaderItem.kt
package vergecurrency.vergewallet.view.ui.components.transaction

import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView

import vergecurrency.vergewallet.R

class TransactionHeaderItem(private val name: String) : TransactionItem {
    override val viewType: Int
        get() =TransactionRowType.HEADER_ITEM.ordinal

    override fun getView(inflater: LayoutInflater, convertView: View?, parent: ViewGroup, position: Int): View {
        val view: View

        if (convertView == null) {
            view = inflater.inflate(R.layout.listview_item_transactions_header, null)
        } else {
            view = convertView
        }
        val txv = view.findViewById<TextView>(R.id.listview_transactions_header_title)
        txv.text = name
        return view
    }

}
